iothub/service/src/DigitalTwin/DigitalTwinClient.cs

```diff
@@ -98,6 +98,7 @@ public static DigitalTwinClient CreateFromConnectionString(string connectionStri
 
         /// <summary>
         /// Creates an instance of <see cref="DigitalTwinClient"/>.
+        /// For more information on configuring IoT hub with Azure Active Directory, <see href="https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-dev-guide-azure-ad-rbac"/>
```

Comment:
Remember that the `<see>` tag does not insert a verb. So this would come out as:
> For more information on configuring IoT hub with Azure Active Directory, https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-dev-guide-azure-ad-rbac

You'll want to add your own verb, and give it puncuation.